optuna/study.py

```diff
@@ -338,6 +336,8 @@ def trials_dataframe(self, include_internal_fields=False):
                 By default, internal fields of :class:`~optuna.structs.FrozenTrial` are excluded
                 from a DataFrame of trials. If this argument is :obj:`True`, they will be included
                 in the DataFrame.
+            multi_index:
+                Specifies whether the returned DataFrame_ employs MultiIndex_ or not.
```

Comment:
I couldn't find the new names of the columns in the document. I think it would be helpful for users if you mention it with examples.

![image](https://user-images.githubusercontent.com/3255979/69781885-d914d380-11f2-11ea-9bb3-002208fd811c.png)

Reply:
Sorry for bothering you, but I changed my mind because the latest document has been updated. It does not have any information about column names, and users can easily confirm the column names by printing dataframes.

Reply:
I made the description of the argument slightly more verbose in https://github.com/optuna/optuna/pull/736/commits/b0c24148453985a64c18fd15e60c9cbc92d216e3.